### PR TITLE
#167531845 Create article card row

### PR DESCRIPTION
**Pull request by akinyeleolat**
#### What does this PR do?

This PR create an article card row component

#### Description of task to be completed?

- setup container for the molecules
-Implement article card row
-Update the style guide
-Write unit test

#### How should this be manually tested?
$ git clone https://github.com/andela/king-kong-ah-frontend.git
$ npm install
$ npm test

#### Any background context you want to provide?

You would need npm to run this application

#### What are the relevant pivotal tracker stories?

PT Story link - # 167531845

#### Screenshots (if appropriate)
Screenshot 2019-07-31 at 12 40 21 PM

### CODE REVIEW

File: src/components/UI/organisms/DashboardWelcome/DashboardWelcome.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import FlexContainer from '<atoms>/layouts/FlexContainer/FlexContainer';
+import Image from '<atoms>/Image/Image';
+import Title from '<atoms>/Title/Title';
+import ImageUrl from '<image>/homepage.png';
+import SearchBar from '<molecules>/SearchBar/SearchBar';
+
+const DashboardWelcome = () => (
```
Comment (mosinmiloluwa01) [inline src/components/UI/organisms/DashboardWelcome/DashboardWelcome.jsx]: i think this should be something like `WriteArticleSection`

**Review by mosinmiloluwa01 — CHANGES_REQUESTED** [review]
Good job so far, but It'll be nice to populate those cards with actual data coming from the back-end

File: src/components/UI/organisms/ArticleCardRow/ArticleCardRow.jsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
+/* eslint-disable no-use-before-define */
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import FlexContainer from '<atoms>/layouts/FlexContainer/FlexContainer';
+import Text from '<atoms>/Text/Text';
+import ArticleCard from '<molecules>/ArticleCard/ArticleCard';
+
+const ArticleCardRow = ({
+  title,
+  articleCardItem,
```
Comment (benfluleck) [inline src/components/UI/organisms/ArticleCardRow/ArticleCardRow.jsx]: You are already in articleCard rename to item

File: src/components/UI/organisms/ArticleCardRow/ArticleCardRow.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    title: 'Heart-beating fun',
+    description: '"Its good to come out of your skin sometimes and experience life... go biking or mountain climbing..."',
+    author: 'Wale Adenuga',
+    rating: 3,
+  },
+
+];
+
+<ArticleCardRow
+        title='Featured Article'
```
Comment (benfluleck) [inline src/components/UI/organisms/ArticleCardRow/ArticleCardRow.md]: tabs are off here

**Review by benfluleck — APPROVED** [review]

**Pull request merged**